examples/jwt/src/backend/app/config.clj

```diff
@@ -0,0 +1,19 @@
+(ns app.config)
+
+;; This def is for readability only
+;; the xiana/jwt config should be passed
+;; as an env variable so the private and public
+;; keys don't need to be tracked by version control.
+
+(def private-key (slurp "resources/_files/jwtRS256.key"))
+
+(def public-key (slurp "resources/_files/jwtRS256.key.pub"))
```

Comment:
These files should be configurable, and you can move the init function to the xiana too.